src/Operations.jl

```diff
@@ -135,7 +135,7 @@ function deps_graph(env::EnvCache, pkgs::Vector{PackageSpec})
                     d = get_or_make(Dict{String,UUID}, dependencies, v)
                     r = get_or_make(Dict{String,VersionSpec}, compatibility, v)
                     q = Dict(u => get_or_make(VersionSpec, r, p) for (p, u) in d)
-                    VERSION in get_or_make(VersionSpec, r, "julia") || continue
+                    # VERSION in get_or_make(VersionSpec, r, "julia") || continue
```

Comment:
@StefanKarpinski

What should we do about this? On julia 0.7 nothing will match because the VersionSpec is upper bounded to julia 0.6 on everything.